mono/metadata/exception.c

```diff
@@ -436,6 +436,7 @@ mono_exception_new_thread_interrupted (MonoError *error)
  * mono_get_exception_thread_interrupted:
  * \returns a new instance of the \c System.Threading.ThreadInterruptedException
  */
+MONO_RT_EXTERNAL_ONLY
```

Comment:
Curious why in a couple of places you put `MONO_RT_EXTERNAL_ONLY` in the C file - they should be in `exception.h`